src/_includes/footer2.html

```diff
@@ -0,0 +1,43 @@
+<footer class="siteFooter2">
+  <div class="siteFooter2-links">
+    <ul class="siteFooter2-links-navList">
+      {% for link in site.data.footer2 %}
+        {% assign href = link.href %}
+        {% if link.landingurl %}
+          {% assign href = (href | prepend: "/" | prepend: site.landingurl) %}
+        {% elsif link.docsurl %}
+          {% assign href = (href | prepend: "/" | prepend: site.docsurl) %}
+        {% elsif link.blogurl %}
+          {% assign href = (href | prepend: "/" | prepend: site.blogurl) %}
+        {% endif %}
+      
+        <li class="siteFooter2-links-navItem">
+          {% if link.github %}
+            <a class="siteFooter2-links-navLink siteFooter2-links-navLink--github" href="{{href}}">
+              {{link.name}}
+              <span class="siteHeader2-navLink-divider">{% svg assets/svg/github-star.svg class="siteFooter2-links-navLink-svg"%}</span>
+              <span>{{link.githubStars}}</span>
+            </a>
+          {% else %}
+            <a class="siteFooter2-links-navLink {% if link.primary %}siteFooter2-links-navLink--primary{% endif %}" href="{{href}}">
+              {{link.name}}
+            </a>
+          {% endif %}
+        </li>
+      {% endfor %}
+    </ul>
+    <ul class="siteFooter2-links-socialList">
+      <li class="siteFooter2-links-socialItem">
+        <a class="siteFooter2-links-socialLink" href="https://twitter.com/tilt_dev">
```

Comment:
can you use {{site.twitter_username}} here? that makes it a bit easier to see all the places it's used

Reply:
Yes! Thanks for catching this.